### ISSUE TITLE
Make list items more compact

### ISSUE THREAD

**Issue by lauracowen** [OpenLiberty/openliberty.io#1265]
In lists (ordered and unordered/bulleted), the items are very vertically spread because they use the default paragraph spacing (I think). It's quite common for CSS to make list items more compact by removing some of the white space between list items so that they take up less space on the page and are easier to read.

eg there's no reason for these lists to take up so much space:

<img width="1033" alt="image" src="https://user-images.githubusercontent.com/3322532/61120036-f6d7db00-a493-11e9-8451-1bba925fa54e.png">

<img width="1037" alt="image" src="https://user-images.githubusercontent.com/3322532/61120078-08b97e00-a494-11e9-9801-dd896171196f.png">

**Comment by ellenwyllie** [comment]
Where is this screenshot from? It looks out of date since we have changed the orange links on the blog to a blue color. The list item spacing is not the same as the default paragraph spacing. We already decreased the spacing for lists but if you think it should be smaller then we can do that.

![Screen Shot 2019-07-15 at 9 40 11 AM](https://user-images.githubusercontent.com/22085207/61224805-dce10700-a6e4-11e9-89f0-f29d67eb235d.png)
